src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  actions.forEach(object => {
+    switch (object.type) {
+      case 'addProperties':
+        for (const key in object.extraData) {
+          state[key] = object.extraData[key];
+        }
+        break;
+      case 'removeProperties':
+        object.keysToRemove.forEach(key => {
+          delete state[key];
+        });
+        break;
+
+      default:
```

Comment:
No, let's define specific action `clear`.
Right now, if I provide some random `action` it will clear the object.